### PR TITLE
Mnt/entrypoints

### PR DESCRIPTION
**Pull request by pauldmccarthy**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pauldmccarthy** [comment]
The process that I've been using to manually update/test `conda-forge` recipes is as follows. Not sure how efficient it is, and it doesn't help with cross platform/pyver issues, but it does the job for most tasks...

# Building/testing/changing conda-forge packages

1. Make sure you have a conda environment with the following packages installed and up to date:
  - `conda-build`
  - `conda-smithy`
  - `conda-forge-pinning`


2. Fork the [`nibabel-feedstock`](https://github.com/conda-forge/nibabel-feedstock) repository, and clone your fork.


3. Create a test environment somewhere:
```
conda create -p testenv python=3.6
conda activate `pwd`/testenv
```


4. In the feedstock repository root, run this to build the package:
```
conda-build --output-folder=./dist ./recipe
```

5. If necessary, uninstall the previous build, and remove the cached package directory:
```
conda uninstall nibabel
rm -r $CONDA_PREFIX_1/pkgs/nibabel-*
```

6. Install the package into your test environment:
```
conda install -c `pwd`/dist nibabel
```


7. Make some changes to the recipe, and repeat steps 4-6 until you are happy.


8. Commit your changes (remember to bump the version/build numbers in `meta.yaml` if necessary).


9. Re-render the recipe (incorporate updates to the `conda-forge` CI machinery). It will make and stage some changes, and will print out a `git commit` command for you to use:
```
conda smithy rerender
git commit -m '...'
```

10. Push the commits to your fork, and then use github to PR them to `conda-forge/nibabel-feedstock`

**Comment by pauldmccarthy** [comment]
@yarikoptic @effigies I couldn't reproduce the issue at nipy/nibabel#717 - would you mind having a play with this to see if it fixes the problem?

**Comment by effigies** [comment]
@yarikoptic I don't have a Windows machine to test on. Can you verify? You may need to build on a Linux system and copy the package to Windows.

**Comment by pauldmccarthy** [comment]
[Here's a build from my laptop](https://github.com/conda-forge/nibabel-feedstock/files/2961963/dist.tar.gz) (the `dist` directory from my above instructions), if you are like me and enjoy downloading and executing random tarballs from the internet...

**Comment by effigies** [comment]
Only if it's signed with the private key of someone I have no reason to trust.

**Comment by yarikoptic** [comment]
> @yarikoptic I don't have a Windows machine to test on. Can you verify? You may need to build on a Linux system and copy the package to Windows.

not that easily - there should be some VMs laying around but every time I am boot it I cry feeling helpless.  You don't want that to your fella neighbor, do you?

But otherwise addition of entry points looks legit

**Comment by yarikoptic** [comment]
what I mean -- easier just to merge, and then try on some windows box, or wait for ppl to report a trouble -- it shouldn't make it worse

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,6 +46,15 @@ requirements:
 test:
   imports:
     - nibabel
+  commands:
+    - command -v parrec2nii
```
Comment (yarikoptic) [inline recipe/meta.yaml]: what does `command -v` do?  I see no mentioning of that in https://conda-forge.readthedocs.io/en/latest/testing.html

Comment (grlee77) [reply recipe/meta.yaml]: It is similar to `which` on linux
http://www.linuxhowtos.org/manpages/1p/command.htm

**Comment by grlee77** [comment]
I built this on a linux machine then copied the .tar.bz2 to an old Windows 7 machine. From there I did:

`conda install nibabel-2.3.3-py_1.tar.bz2`

It does have `parrec2nii.exe`, `nib-ls.exe`, etc. in the Scripts subfolder of the environment and they seem to work okay.

File: recipe/meta.yaml (hunk 1 above)
Comment (pauldmccarthy) [reply recipe/meta.yaml]: Similar to `which`, but reliably returns a non-zero exit code if the commnad doesn't exist: https://stackoverflow.com/a/677212

**Comment by pauldmccarthy** [comment]
As suspected, `command -v <cmd>` does not work under windows. But `WHERE <cmd>` does appear to work.  Shall I make the tests future-proof, in the event that `nibabel` may need to be built for separate platforms in the future?

**Comment by yarikoptic** [comment]
Went don't you just run those commands with --help, or you really want to see where they are installed?

**Comment by pauldmccarthy** [comment]
@yarikoptic that would be much better - I tried just calling each command without any args, but got inconsistent exit codes. I didn't try `--help` though!

**Comment by effigies** [comment]
Everybody happy with this? It doesn't look like it'll hurt, at worst.

**Comment by grlee77** [comment]
sure. let's try it

**Pull request merged**